x-pack/packages/ml/date_picker/src/components/full_time_range_selector.tsx

```diff
@@ -123,10 +123,9 @@ export const FullTimeRangeSelector: FC<FullTimeRangeSelectorProps> = (props) =>
   const setPreference = useCallback(
     (id: string) => {
       setFrozenDataPreference(id as FrozenTierPreference);
-      setRange(dataView, query, id === FROZEN_TIER_PREFERENCE.EXCLUDE);
       closePopover();
     },
-    [dataView, query, setFrozenDataPreference, setRange]
+    [setFrozenDataPreference]
```

Comment:
This doesn't appear to be working correctly.
No matter what I choose in the dropdown, the search fired off contains the bool query to avoid frozen data.
```
"must_not": [
  {
    "term": {
      "_tier": {
        "value": "data_frozen"
      }
    }
  }
]
```

Reply:
Thanks for catching this, I updated the behaviour in [c47e1a3](https://github.com/elastic/kibana/pull/151673/commits/c47e1a30fa01ca7ce41d74399e6c2b59b6fa8181).